README.md

```diff
@@ -34,6 +34,8 @@ let g:keepeye_autostart = v:true                       " launch at vim startup
 let g:keepeye_callback = 'keepeye#Callback'            " customize the callback
 let g:keepeye_message = 'SAVE YOUR EYES, TAKE A BREAK' " set the message to show
 let g:keepeye_system_bell = v:false                    " enable the bell sound (1)
+let g:keepeye_bell_cmd = 'vclc'                        " set a custom player to ring the bell
+let g:keepeye_bell_cmd_flags = 'flags'                 " set the player flags to suit your needs
```

Comment:
I would rename both to `g:keepeye_system_bell_*`, to match with `g:keepeye_system_bell`. The name starts to be long, but it's better imo. This plugin starts to have a lot of options, I'm thinking about compact them in the future.